lib/danger/request_sources/github.rb

```diff
@@ -30,6 +30,13 @@ def markdown_parser
     def fetch_details
       self.pr_json = client.pull_request(ci_source.repo_slug, ci_source.pull_request_id)
       fetch_issue_details(self.pr_json)
+      self.ignored_violations = ignored_violations_from_pr(self.pr_json)
+    end
+
+    def ignored_violations_from_pr(pr_json)
+      pr_body = pr_json[:body]
+      return [] if pr_body.nil?
+      pr_body.chomp.scan(/>\s*danger\s*:\s*ignore\s*"(.*)"/i).flatten
```

Comment:
I can change that, but it shouldn't matter unless someone triggers an empty warning/error